src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/PreferDictionaryTryMethodsOverContainsKeyGuardAnalyzer.cs

```diff
@@ -177,74 +265,92 @@ private static bool GetParentConditionalOperation(IOperation operation, ref Dict
                 {
                     case IConditionalOperation c:
                         conditionalOperation = c;
+
                         return true;
                     case IBinaryOperation { OperatorKind: BinaryOperatorKind.ConditionalAnd } bAnd when guardsTruePath:
                         nextOperation = bAnd;
-                        if (bAnd.LeftOperand == operation && !FindUsages(bAnd.RightOperand, ref context))
+                        if (bAnd.LeftOperand == operation && !FindUsages(bAnd.RightOperand, ref usageContext, searchContext))
                         {
                             conditionalOperation = null;
+
                             return false;
                         }
 
                         break;
                     case IBinaryOperation { OperatorKind: BinaryOperatorKind.ConditionalOr } bOr when !guardsTruePath:
                         nextOperation = bOr;
-                        if (bOr.LeftOperand == operation && !FindUsages(bOr.RightOperand, ref context))
+                        if (bOr.LeftOperand == operation && !FindUsages(bOr.RightOperand, ref usageContext, searchContext))
                         {
                             conditionalOperation = null;
+
                             return false;
                         }
 
                         break;
 
                     default:
                         conditionalOperation = null;
+
```

Comment:
NIT: There is lots of extra line added in switch cases, here an below

Reply:
I believe that is due to `IDE2003`. But I removed the blank lines where possible.